lib/elixir/test/elixir/collectable_test.exs

```diff
@@ -4,4 +4,20 @@ defmodule CollectableTest do
   use ExUnit.Case, async: true
 
   doctest Collectable
+
+  setup do
+    {map_acc, _map_fun} = Collectable.Map.into(%{a: 1})
+    {empty_map_acc, map_fun} = Collectable.Map.into(%{})
+    %{map_acc: map_acc, empty_map_acc: empty_map_acc, map_fun: map_fun}
```

Comment:
There is no need to use setup here, as `setup` is shared across tests, and this is likely the only test that will use these exact same vars. :)

I also think testing a single input is fine. No need to test both empty maps and non empty ones.